src/rest/resource.pug

```diff
@@ -199,12 +199,24 @@ mixin resourceModals(resource)
 mixin resourceRows(resource)
     if resource.methods
         for method in resource.methods
-            tr.rest-endpoint(onclick=`window.location.href = '#${resource.uniqueId}_${method.method}'`)
+            if method.annotations && method.annotations.deprecated
+                - deprecated = true
+            else
+                - deprecated = false
```

Comment:
i don't really know pug but can you not do `deprecated = method.annotations && method.annotations.deprecated` ?